service/src/main/java/aero/minova/cas/controller/SqlProcedureController.java

```diff
@@ -235,6 +266,38 @@ public SqlProcedureResult unsecurelyProcessProcedure(Table inputTable) throws Ex
 		return processSqlProcedureRequest(inputTable, authority);
 	}
 
+	/**
+	 * Diese Methode ist nicht geschützt. Aufrufer sind für die Sicherheit verantwortlich. Führt eine Prozedur mit den übergebenen Parametern aus. Falls die
+	 * Prozedur Output-Parameter zurückgibt, werden diese auch im SqlProcedureResult zurückgegeben. Prüft auch, ob eine Extension statt einer Prozedur
+	 * aufgerufen werden muss.
+	 *
+	 * @param inputTable
+	 *            Ausführungs-Parameter im Form einer Table
+	 * @return Optional einer SqlProcedureResult der Ausführung
+	 * @throws Exception
+	 *             Fehler beim Ausführen der Prozedur.
+	 */
+	public ResponseEntity<?> unsecurelyProcessProcedureWithExtensionCheck(Table inputTable) throws Exception {
```

Comment:
Wie wäre es mit unsecurelyExecuteProcedure in Anlehnung an executeProcedure,
welche die selbe Funktion haben soll, wenn man vom Rechtesystem absieht.

Damit wäre es auch kürzer.